apps/server/src/adapters/WebsocketAdapter.ts

```diff
@@ -135,12 +180,26 @@ export class SocketServer implements IAdapter {
     });
   }
 
+  sendClientList(): void {
+    const payload = Array.from(this.clients.entries()).reduce((acc, [key, data]) => {
+      //We have to remove the WS part from the list before sending it to the clients
+      const { ws: _, ...dataExcludingWs } = data;
+      return Object.assign(acc, { [key]: dataExcludingWs });
+    }, {});
+    this.sendAsJson({ type: 'client-list', payload });
+  }
```

Comment:
I think this logic could be hard to maintain in the long run. Few suggestions:
- since the size of the thing coming in is the same as the ones going out, a map may be easier to leverage
- we dont need to cast the `ws` to `_` just to silence eslint
- could we add some comments about what does this do and why do we need to exclude ws?

Something like (untested)

```ts
    const payload = Object.fromEntries(
        Array.from(this.clients.entries()).map(([key, data]) => {
            // eslint-disable-next-line @typescript-eslint/no-unused-vars -- using destructuring to remove property
            const { ws, ...dataExcludingWs } = data;
            return [key, dataExcludingWs];
        })
    );
    this.sendAsJson({ type: 'client-list', payload });
```